tests/sync_diff_inspector/time_zone/run.sh

```diff
@@ -0,0 +1,33 @@
+
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")"
+
+OUT_DIR=/tmp/tidb_tools_test/sync_diff_inspector
+
+
+mysql -uroot -h 127.0.0.1 -P 4000 -e "create database if not exists tz_test"
+mysql -uroot -h 127.0.0.1 -P 4000 -e "create table tz_test.diff(id int, dt datetime, ts timestamp);"
+mysql -uroot -h 127.0.0.1 -P 4000 -e "insert into table tz_test.diff values (1, now(), now());"
+
+echo "dump data and then load to tidb"
+rm -rf $OUT_DIR/dump_tz_diff
+mydumper --host 127.0.0.1 --port 4000 --user root --outputdir $OUT_DIR/dump_tz_diff -B tz_test -T diff
+loader -h 127.0.0.1 -P 4001 -u root -d $OUT_DIR/dump_tz_diff
+
+echo "check with the same time_zone, check result should be pass"
+sync_diff_inspector --config=./config.toml > $OUT_DIR/ignore_column_diff.log || true
```

Comment:
```suggestion
sync_diff_inspector --config=./config.toml > $OUT_DIR/different_timezone_diff.log || true
```

why there's `|| true`